designs/2022-supress-ignored-file-warnings/README.md

```diff
@@ -143,6 +222,8 @@ It was considered to output the warning to `stderr` instead, but that would caus
 
 ESLint [collects suppressed warnings and errors](https://github.com/eslint/eslint/pull/15459). In case `--no-warning-on-ignored-files` or `warnIgnored: false` is used, should the warning be included in the suppressed messages array?
 
+Should the `warnIgnored` option be available via `eslint.config.js` too?
```

Comment:
My vote is no, let's keep it as a CLI only option.

Reply:
Agreed, this doesn't seem like an option that should be available in eslint config files.